vernac/metasyntax.ml

```diff
@@ -83,7 +83,10 @@ let pr_grammar = function
       pr_entry Pvernac.Vernac_.gallina ++
       str "Entry gallina_ext is" ++ fnl () ++
       pr_entry Pvernac.Vernac_.gallina_ext
-  | name -> pr_registered_grammar name
+  | _ -> user_err Pp.(str "Unknown grammar entry." ++ spc()
+                      ++ str "Use Print Custom Grammar to inspect custom entries.")
```

Comment:
Perfect :+1: That's what I was asking for!

Reply:
Note that this change got reverted (it was clobbering `Print Grammar tactic`), but there is still a check in `pr_registered_grammar`.